src/core/MOM_dynamics_split_RK2.F90

```diff
@@ -1228,6 +1263,10 @@ subroutine initialize_dyn_split_RK2(u, v, h, uh, vh, eta, Time, G, GV, US, param
       'Zonal Pressure Force Acceleration', 'm s-2', conversion=US%L_T2_to_m_s2)
   CS%id_PFv = register_diag_field('ocean_model', 'PFv', diag%axesCvL, Time, &
       'Meridional Pressure Force Acceleration', 'm s-2', conversion=US%L_T2_to_m_s2)
+  CS%id_hfPFu = register_diag_field('ocean_model', 'hf_PFu', diag%axesCuL, Time, &
```

Comment:
I think the new diagnostics probably should have `v_extensive = .true.` because they are thickness weighted.

Reply:
I have made this change but need to confirm if` v_extensive = .true.` is correct in this case.